install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml

```diff
@@ -23,8 +23,6 @@ rules:
   - watch
 - apiGroups: ["config.istio.io"]
   resources:
-  - routerules
-  - destinationpolicies
```

Comment:
Why delete `destinationpolicies`?

Reply:
kiali at this point is not accessing routerules or destinationpolicies - we are using the new stuff, so we don't need these in our cluster roles.

Reply:
@jmazzitelli , @gbaufake does the Kiali document highlight this? With 0.5.0, Kiali will not able to view routerules?

Also why not show destinationpolicies? This is still used in istio.